main.go

```diff
@@ -0,0 +1,200 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"net/http/cookiejar"
+	"os"
+	"time"
+
+	"github.com/xuri/excelize/v2"
+)
+
+const (
+	maxPasswordAgeDays int = 30
+)
+
+type Input struct {
+	Filename                     string
+	SheetName                    string
+	UsernameHeader               string
+	PasswordHeader               string
+	AutomatedSheetPassword       string
+	automatedSheetName           string // sheet managed by application
+	automatedSheetColNameToIndex map[string]int
```

Comment:
how about

```
type Column int

const (
  ColUser Column = iota
  // etc
)
```

and then 
```
automatedSheetColNameToIndex map[Column]int
```


(it's just for convenience that `int` is the type chosen for `Column`, it's not because it will be mapped to indexes.)

Reply:
I was wondering how to use the const with iota now that I moved the map of cols to indexes to Input. Clean and simple. Thank you. Done.